src/rw-widgets.js

```diff
@@ -18,4 +19,29 @@ BeatBlocks.addWidgetToRegistry('river', RiverWidget);
 BeatBlocks.addWidgetToRegistry('timeline', TimelineWidget);
 BeatBlocks.addWidgetToRegistry('financial', FinancialWidget);
 
+// The Numeral library allows whole-language overrides. Using that to map our
+// unit abbreviations to longer terms.
+numeral.language('en-long', {
+  delimiters: {
+    thousands: ',',
+    decimal: '.'
+  },
+  abbreviations: {
+    thousand: 'thousand',
+    million: 'million',
+    billion: 'billion',
+    trillion: 'trillion'
+  },
+  ordinal: function (number) {
+    var b = number % 10;
+    return (~~ (number % 100 / 10) === 1) ? 'th' :
+      (b === 1) ? 'st' :
+        (b === 2) ? 'nd' :
+          (b === 3) ? 'rd' : 'th';
+  },
+  currency: {
+    symbol: '$'
+  }
+});
+
```

Comment:
Numeral has good localization support, which we are leveraging to define a new language that uses full-word names for number sizes in place of the single-letter abbreviations used by default.

`numeric.language('en')` and `numeric.language('en-long')` can be toggled in between uses of `numeric(value).format('0.0')` to format values differently.